workflows/mhcquant.nf

```diff
@@ -0,0 +1,405 @@
+include { OPENMS_THERMORAWFILEPARSER }                      from '../modules/local/process/openms_thermorawfileparser'                       addParams( options: [:] )
+include { OPENMS_PEAKPICKERHIRES }                          from '../modules/local/process/openms_peakpickerhires'                           addParams( options: [:] )
+include { OPENMS_COMETADAPTER }                             from '../modules/local/process/openms_cometadapter'                              addParams( options: openms_comet_adapter_options )
+include { OPENMS_PEPTIDEINDEXER }                           from '../modules/local/process/openms_peptideindexer'                            addParams( options: [:] )
+include { OPENMS_FALSEDISCOVERYRATE }                       from '../modules/local/process/openms_falsediscoveryrate'                        addParams( options: [:] )
+include { OPENMS_IDFILTER as OPENMS_IDFILTER_FOR_ALIGNMENT }from '../modules/local/process/openms_idfilter'                                  addParams( options: id_filter_options )
+include { OPENMS_IDFILTER as OPENMS_IDFILTER_Q_VALUE }      from '../modules/local/process/openms_idfilter'                                  addParams( options: id_filter_qvalue_options )
+include { OPENMS_MAPALIGNERIDENTIFICATION }                 from '../modules/local/process/openms_mapaligneridentification'                  addParams( options: openms_map_aligner_identification_options )
+
+include { 
+    OPENMS_MAPRTTRANSFORMER as OPENMS_MAPRTTRANSFORMERMZML
+    OPENMS_MAPRTTRANSFORMER as OPENMS_MAPRTTRANSFORMERIDXML }        from '../modules/local/process/openms_maprttransformer'                 addParams( options: [:] )
+
+include { OPENMS_IDMERGER }                                 from '../modules/local/process/openms_idmerger'                                  addParams( options: [:] )
+include { OPENMS_PSMFEATUREEXTRACTOR }                      from '../modules/local/process/openms_psmfeatureextractor'                       addParams( options: [:] )
+include { OPENMS_PERCOLATORADAPTER }                        from '../modules/local/process/openms_percolatoradapter'                         addParams( options: percolator_adapter_options )
+include { OPENMS_PERCOLATORADAPTER as OPENMS_PERCOLATORADAPTER_KLAMMER } from '../modules/local/process/openms_percolatoradapter'            addParams( options: percolator_adapter_klammer_options )
+
+include { REFINE_FDR_ON_PREDICTED_SUBSET }                  from '../modules/local/subworkflow/refine_fdr_on_predicted_subset'               addParams( run_percolator_options : percolator_adapter_options, filter_options: id_filter_options, whitelist_filter_options: id_filter_whitelist_options)
+
+include { OPENMS_FEATUREFINDERIDENTIFICATION }              from '../modules/local/process/openms_featurefinderidentification'               addParams( options: [:] )
```

Comment:
Maybe I am overlooking sth., but there should be an IDFilter after Percolator and before FeatureFinderIdentification outside of the REFINE_FDR_ON_PREDICTED_SUBSET option...

Reply:
There were 5 IDFilter steps but step 12 and 12.0 are merged. The other IDFilter steps are: 
1. filter_fdr_for_idalignment > OPENMS_IDFILTER_FOR_ALIGNMENT
2. filter_by_q_value & filter_by_q_value_first > OPENMS_IDFILTER_Q_VALUE
3. filter_psms_by_predictions > OPENMS_IDFILTER_PSMS
4. filter_refined_q_value > OPENMS_IDFILTER_REFINED

Reply:
ok then all good